snippets/language-go.cson

```diff
@@ -107,3 +107,6 @@
   'goroutine function':
     'prefix': 'gf'
     'body': 'go ${1:func}($0)'
+  'test':
+    'prefix': 'test'
```

Comment:
And to keep things a little inline with each other, could we change this to `tf`?